tests/scalability/scale_test_runner.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import datetime
+import os
+import sys
+import time
+import uuid
+
+import argparse
+
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), 'sns')))  # noqa
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '../..')))  # noqa
+
+from sns import SnsClient
+
+UNIT_OF_TIME = 1 * 1000 * 1000  # 1 sec in microseconds
```

Comment:
Constants should be called something more descriptive. Perhaps `ONE_SECOND_IN_MS`

Reply:
done